softdevice_controller/include/sdc_hci_cmd_controller_baseband.h

```diff
@@ -336,14 +336,14 @@ uint8_t sdc_hci_cmd_cb_read_transmit_power_level(const sdc_hci_cmd_cb_read_trans
  * off, the Host should not send the HCI_Host_Number_Of_Completed_Packets
  * command. That command will be ignored by the Controller if it is sent by the
  * Host and flow control is off. If flow control is turned on for HCI ACL Data packets
- * and off for HCI Synchronous Data packets, HCI_Host_Number_Of_Completed_-
- * Packets commands sent by the Host should only contain Connection_Handles
+ * and off for HCI Synchronous Data packets, HCI_Host_Number_Of_Completed_Packets
```

Comment:
This file and similar header files are autogenerated from a PDF document. I would therefore prefer to keep them as is

Reply:
@rugeGerritsen : Is it possible to fix this formatting issues in the PDF. Presently the commands render in the following format:

![image](https://user-images.githubusercontent.com/91891495/175496901-30bce8ba-6ecd-4d4a-aca3-445f1f5a0ee4.png)

I would be reverting back for following header files. Hope these are the files that should be considered:
softdevice_controller/include/sdc_hci_cmd_controller_baseband.h
softdevice_controller/include/sdc_hci_cmd_le.h
softdevice_controller/include/sdc_hci_cmd_status_params.h

Reply:
The PDF is the Bluetooth Core specification, so that is not something we can change. But we may look into if we can change how we parse the PDF. In any case, I would prefer to leave that out of this PR